### PR TITLE
.github: Upgrade windows CUDA 10.1 -> 10.2

### PR DESCRIPTION
**Pull request by seemethere**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #64658

We don't release 10.1 anymore so let's bump to 10.2

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D30811178](https://our.internmc.facebook.com/intern/diff/D30811178)

cc @peterjc123 @mszhanyi @skyline75489 @nbcsm @ezyang @seemethere @malfet @lg20987 @pytorch/pytorch-dev-infra

### CODE REVIEW

**Comment by seemethere** [comment]
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30811178).

**Review by malfet — APPROVED** [review]

**Comment by seemethere** [comment]
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30811178).

**Review by janeyx99 — APPROVED** [review]

File: .circleci/scripts/windows_cuda_install.sh [hunk 1]
```diff
@@ -4,9 +4,9 @@ set -eux -o pipefail
 cuda_major_version=${CUDA_VERSION%.*}
 
 if [[ "$cuda_major_version" == "10" ]]; then
-    cuda_installer_name="cuda_10.1.243_426.00_win10"
+    cuda_installer_name="cuda_10.2.89_441.22_win10"
     msbuild_project_dir="CUDAVisualStudioIntegration/extras/visual_studio_integration/MSBuildExtensions"
-    cuda_install_packages="nvcc_10.1 cuobjdump_10.1 nvprune_10.1 cupti_10.1 cublas_10.1 cublas_dev_10.1 cudart_10.1 cufft_10.1 cufft_dev_10.1 curand_10.1 curand_dev_10.1 cusolver_10.1 cusolver_dev_10.1 cusparse_10.1 cusparse_dev_10.1 nvgraph_10.1 nvgraph_dev_10.1 npp_10.1 npp_dev_10.1 nvrtc_10.1 nvrtc_dev_10.1 nvml_dev_10.1"
+    cuda_install_packages="nvcc_10.2 cuobjdump_10.2 nvprune_10.2 cupti_10.2 cublas_10.2 cublas_dev_10.2 cudart_10.2 cufft_10.2 cufft_dev_10.2 curand_10.2 curand_dev_10.2 cusolver_10.2 cusolver_dev_10.2 cusparse_10.2 cusparse_dev_10.2 nvgraph_10.2 nvgraph_dev_10.2 npp_10.2 npp_dev_10.2 nvrtc_10.2 nvrtc_dev_10.2 nvml_dev_10.2"
```
Comment (malfet) [inline .circleci/scripts/windows_cuda_install.sh:9]: Can we keep both 10.1 and 10.2 versions here. Also, you'll need to update cudnn download link as well

**Comment by seemethere** [comment]
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30811178).

**Pull request closed without merging**